src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Pledges.cshtml

```diff
@@ -34,6 +35,7 @@
                             <th scope="col" class="govuk-table__header govuk-table__header--numeric">Total</th>
                             <th scope="col" class="govuk-table__header govuk-table__header--numeric">Remaining funds</th>
                             <th scope="col" class="govuk-table__header govuk-table__header--numeric">Applications</th>
+                            <th scope="col" class="govuk-table__header govuk-table__header--numeric">Status</th>
```

Comment:
I have seen just govuk-table__header being used for text fields at other places.  May be we could do the same here?

Reply:
Sure, this is now updated.